packages/proto-signing/src/pubkey.spec.ts

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import { fromBase64 } from "@cosmjs/encoding";
+
+import { google } from "./codec";
+import { decodePubkey, encodePubkey } from "./pubkey";
+
+const { Any } = google.protobuf;
+
+describe("pubkey", () => {
+  const defaultPubkeyBase64 = "AtQaCqFnshaZQp6rIkvAPyzThvCvXSDO+9AzbxVErqJP";
+  const defaultPubkeyBytes = fromBase64(defaultPubkeyBase64);
+  const defaultPubkeyProtoBytes = Uint8Array.from([0x0a, defaultPubkeyBytes.length, ...defaultPubkeyBytes]);
```

Comment:
Any clue what kind of encoding this is, especially the leading `0x0a`?

Reply:
I believe this is protobuf for "field 1; length-prefixed".

See https://github.com/protobufjs/protobuf.js/wiki/How-to-reverse-engineer-a-buffer-by-hand#so-what-are-those-wire-types

Reply:
Ah, then this is https://github.com/cosmos/cosmos-sdk/blob/v0.40.0-rc0/proto/cosmos/crypto/secp256k1/keys.proto#L13